docs/examples.md

```diff
@@ -138,8 +137,9 @@ bitcoin-cli scantxoutset start '[{"desc":"wpkh(xpub6DP9afdc7qsz7s7mwAvciAR2dV6vP
 ```
 
 
-[1]: https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md
-[2]: https://github.com/bitcoin/bips/blob/master/bip-0049.mediawiki
-[3]: https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki
+## Notes
+
+<a name="bitcoin-core-version" title="Note on Bitcoin Core version">1</a> "Bitcoin Core" refers to v0.17.x and later.
```

Comment:
Please don't use html. Anchors like these should be doable with just markdown. Also, I'm not sure that we really need these anchors.

Reply:
> Please don't use html. Anchors like these should be doable with just markdown.

Unfortunatly GitHub Flavored Markdown [doesn't support footnotes](https://stackoverflow.com/a/25584585/1117929) (named anchors)

> Also, I'm not sure that we really need these anchors.

The idea was to reduce redundancy, but I can remove if preferred and just use inline links.

Reply:
@achow101 Do you want to leave this as-is or should I convert to inline links to the info?

Reply:
Can you do inline links?